ruby-base/Dockerfile

```diff
@@ -53,6 +53,7 @@ RUN apt-get update -y && \
         libgdbm-dev \
         libgit2-dev \
         libgmp-dev \
+        libjemalloc-dev \
```

Comment:
@brianburnszd we probably also need `libjemalloc1` package here in order to compile Ruby with jemalloc